README.md

```diff
@@ -1,2 +1,183 @@
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://webpack.js.org/guides/getting-started/#basic-setup">Webpack Setup</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[Header section - Introduction]**
+- **[Programs section - Project description]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+ ## 🚀 Live Demo <a name="live-demo"></a> 
+ 
+ [Live Demo Link](https://RKanyoro.github.io/To-DoList/)
+ 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- A code editor of your choice (like vs code or atom and so on)
+-version control system (git is preferred)
+
+### Setup
+
+- Create a new folder to store the files in.
+
+### Install
+
+Install this project with:
+
+- Open your command prompt or terminal and run.
+-  git clone https://github.com/RKanyoro/AwesomeBooks-ES6.git
+
+### Usage
+
+- this is my portflio and will use to insert my upcoming projects.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+- npm start
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+- [🚀 Live Demo Link Provided Above ⬆️](#live-demo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@RKanyoro](https://github.com/RKanyoro)
+- Twitter: [@CarlosMbugua](https://twitter.com/CarlosMbugua)
+- LinkedIn: [Robert Mbugua](https://www.linkedin.com/in/robert-mbugua-a8745093/)
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added:
+
+- [ ] **[Project: Application deploy]**
+- [ ] **[Project: Home (Pair-programming)]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Microverse materials
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose. Kindly ensure that MIT license is working to ensure that README file is professional ☺️.